modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-feature-search/ddm-template.ftl

```diff
@@ -0,0 +1,399 @@
+		line-height: 1rem;
+		text-align: center;
+	}
+
+	.pagination-bar .dropdown.pagination-items-per-page .dropdown-toggle.page-link .c-inner {
+		align-items: center;
+		display: flex;
+		height: 1rem;
+		justify-content: center;
+	}
+
+	.pagination-bar .dropdown.pagination-items-per-page .dropdown-toggle.page-link .c-inner .lexicon-icon.lexicon-icon-caret-double-l {
+		align-items: center;
+		display: flex;
+		flex-shrink: 0;
+		height: 1rem;
+		justify-content: center;
+		width: 1rem;
+	}
+
+	.pagination-bar .pagination {
+		align-items: flex-start;
+		display: flex;
+		gap: 0.5rem;
+		justify-content: center;
+	}
+
+	.pagination-bar .pagination .page-item .page-link {
+		align-items: center;
+		border-radius: 0.375rem;
+		border-width: 0rem;
+		color: var(--color-action-neutral-default, #2B3A4B);
+		display: flex;
+		font-family: 'Source Sans Pro', sans-serif;
+		font-size: 0.875rem;
+		font-style: normal;
+		font-weight: 600;
+		gap: 0.25rem;
+		justify-content: center;
+		line-height: 1rem;
+		min-width: 2rem;
+		padding: 0.5rem;
+		text-align: center;
+	}
+
+	.pagination-bar .pagination .page-item.active .page-link {
+		background: var(--color-action-neutral-active-20, #D5D8DB);
+	}
+
+	.pagination-bar .pagination-results {
+		align-items: flex-start;
+		color: var(--color-neutral-10, #282934);
+		display: flex;
+		font-family: 'Source Sans Pro', sans-serif;
+		font-size: 0.8125rem;
+		font-style: normal;
+		font-weight: 400;
+		gap: 0.25rem;
+		line-height: 1rem;
+		margin-top: 0.5rem;
+	}
+
+	.search-icon>svg {
+		fill: #0B5FFF;
+		height: 2rem;
+		margin: 1.5rem 3.5rem;
+		width: 2rem;
+	}
+
+	.search-results .search-results-entry {
+		align-items: flex-start;
+		display: flex;
+		flex-direction: column;
+		gap: 0.25rem;
+	}
+
+	.search-results .search-results-entry .search-results-entry-title {
+		color: var(--color-neutral-10, #282934);
+		border-radius: 0.625rem;
+		font-family: 'Source Sans Pro', sans-serif;
+		font-size: 1.125rem;
+		font-style: normal;
+		font-weight: 600 !important;
+		line-height: 1.25rem;
+		padding: 1rem 1rem 1rem 0.625rem;
+	}
+
+	.search-results .search-results-entry .search-results-entry-title .search-results-entry-content {
+		color: var(--color-neutral-10, #282934);
+		font-family: 'Source Sans Pro', sans-serif;
+		font-size: 1rem;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 1.5rem;
+		margin-top: 1rem;
+	}
+
+	.search-results .search-results-entry .search-results-entry-title:hover {
+		background: var(--color-action-primary-hover-10, #EDF3FE);
+	}
+
+	.search-results .solid {
+		border-top: 0.0625rem solid var(--color-neutral-2, #E2E2E4);
+	}
+
+	.tooltip-inner a:hover, a.hover {
+		color: var(--color-neutral-0, #FFFFFF);
+		font-weight: var(--font-weight-bold, 700) !important;
+	}
+</style>
+
+<div class="pt-3 search-results" id="searchResults">
+	<#if entries?has_content>
+		<#list entries as searchEntry>
+			<#assign
+				searchEntryTitle = searchEntry.getTitle()!""
```

Comment:
If we are relying on most of our content coming from the rest call (restArticle) it may make sense to also pull the titl from there and make sure the null checks are done on the rest response, not the article title